app/views/vignettes/slides/_form_question.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= form.fields_for :question do |question_form| %>
+  <div>
+    <%= question_form.label :type, "Question Type" %>
+    <%= question_form.select :type, options_for_select([
+        ['No Question', ''],
```

Comment:
I am able to successfully create a slide that does not contain any question. Is this something we want to allow?